scenario/SupplyScenario.ts

```diff
@@ -295,15 +295,14 @@ scenario(
     await albert.allow(betty, true);
     await collateralAsset.approve(albert, betty, 10n * scale);
 
-    await expectRevertMatches(
+    await expect(
       betty.supplyAssetFrom({
         src: albert.address,
         dst: betty.address,
         asset: collateralAsset.address,
         amount: 100n * scale,
-      }),
-      [/ERC20: transfer amount exceeds allowance/, /transfer amount exceeds spender allowance/]
-    );
+      })
+    ).to.be.reverted;
```

Comment:
These scenarios are great, but they assume that the token returns an error message of some sort. LINK reverts without a reason string for these two instances.

Reply:
Oy